pkg/helm/chartaccess.go

```diff
@@ -50,17 +50,23 @@ type chartAccess struct {
 
 var _ ChartAccess = (*chartAccess)(nil)
 
+func newChartAccess(path string, fss ...vfs.FileSystem) (*chartAccess, error) {
```

Comment:
What is the purpose of this extracted function? It isn't called anywhere else other than the temp access below which already does the thing this does. :) Is there a plan to call this from something else?

Reply:
Neat. Thanks.

Reply:
Initially, we didn't realise we had access to the file system of the chart access from the downloader. Therefore, we thought we needed an option to pass in an arbitrary file system. This is pretty much a remnant of that original thought. As it is currently not necessary, I removed it for now.